runtime/gc_base/ScavengerForwardedHeader.hpp

```diff
@@ -306,140 +86,7 @@ class MM_ScavengerForwardedHeader
 	{
 	}
 
-	/**
```

Comment:
Did compiler forced you do have a constructor? Semantically, we (neither native nor DDR) don't need it .